tests/IdentifyServiceProviderTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Unicodeveloper\Identify\Test;
+
+use Mockery\MockInterface;
+use PHPUnit_Framework_TestCase;
+use Unicodeveloper\Identify\IdentifyServiceProvider;
+
+class IdentifyServiceProviderTest extends PHPUnit_Framework_TestCase
+{
+
+    /**
+     * @var MockInterface
+     */
+    protected $mockApp;
+
+    /**
+     * @var IdentifyServiceProvider
+     */
+    protected $provider;
+
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->mockApp = \Mockery::mock();
```

Comment:
You are creating an anonymous mock here. Nice!!

Please can you make an alias of Mockery at the beginning of the file and just use it here, like so:

```php

use Mockery as m;

....
....

$this->mockApp = m::mock();
```